.github/workflows/on-demand-contentful-restore.yml

```diff
@@ -0,0 +1,48 @@
+name: Restore Contentful from backup
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        required: true
+        description: Backup environment to restore
+        type: choice
+        options:
+          - Production
+          - Development
```

Comment:
would we ever want to restore into a dedicated PR environment?

Reply:
Perhaps, and that is possible. The contentful environment is a separate input to be able to support exactly that.

It's possible that these options need to be made clearer, but this input is a selection of the _source_ of the backup - and perhaps more importantly selects which of the [environment config files](https://github.com/yldio/asap-hub/tree/master/.github/environment) to use.

Reply:
What we _might_ want to do is make the contentful environment optional and default to the source environment. It would mean creating a dependency that the contentful environments are always named exactly as the github workflow environments, but that might be tolerable.